reg_form.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <form action="link_for_form" method="post">
+        <fieldset>
+            <legend>Basic User Information:</legend>
+            <label for="Fullname"> Full name:</label>
+            <input type="text" id="Fullname" name="Fullname" required>
+            <br>
+            <label for="Email"> Email:</label>
+            <input type="email" id="Email" name="Email" required>
+            <br>
+            <label for="Password"> Password:</label>
+            <input type="password" id="Password" name="Password" required>
+            <br>
+            <label for="Confirmpassword"> Confirm password:</label>
+            <input type="password" id="Confirmpassword" name="Confirmpassword">
+            <br>
+            <label for="Gender1"> Gender:</label>
+            <input type="radio" id="Gender1" name="Gender" value="male">Male
+            <label for="Gender2"></label>
+            <input type="radio" id="Gender2" value="female">Female
+            <label for="Gender3"></label>
+            <input type="radio" id="Gender3" value="mechanic">My gender...I'm a mechanic
+            <label for="Gender4"></label>
+            <input type="radio" id="Gender4" value="other">Other
+            <br>
+            <label for="DateofBirth"> Date of Birth:</label>
+            <input type="date" id="DateofBirth" name="DateofBirth">
+        </fieldset>
+
+        <fieldset>
+            <legend>Address Information:</legend>
+            <label for="StreetAddress"> Street Addres:</label>
+            <textarea name="StreetAddress" id="StreetAddress" cols="25" rows="2">
+            </textarea>
+            <br>
+            <label for="Country"> Country:</label>
+            <select id="Country" name="Country" size="1">
+                <option value="usa">USA</option>
+                <option value="germany">Germany</option>
+                <option value="russia">Russia</option>
+                <option value="italy">Italy</option>
+                <option value="moldova">Moldova</option>
+            </select>
+            <br>
+            <label for="City"> City:</label>
+            <input type="text" id="City" name="City">
+            <br>
+            <label for="PostalCode"> Postal Code:</label>
+            <input type="number" id="PostalCode" name="PostalCode">
+        </fieldset>
+        <fieldset>
+            <legend>Preferences:</legend>
+            <label for="PreferredContactMethod1"> Preferred Contact Method:</label>
```

Comment:
avoid using CamelCase in classes and ids namings. 
It is advised to use kebab-case or snake_case for naming in HTML and CSS